build.ps1

```diff
@@ -212,16 +212,16 @@ Foreach ($target in $targets) {
         "Migrator" = $([io.path]::combine($specOutDir, "migrator"));
     }
 
-    ValidateRuntimeConfig $target $specOutDirs.Server
+    #ValidateRuntimeConfig $target $specOutDirs.Server
 
     if ($target.Name -eq "windows-x64") {
-        Validate-AssemblyVersion $(Join-Path -Path $specOutDirs.Server -ChildPath "Raven.Server.dll" ) $versionObj
-        Validate-AssemblyVersion $(Join-Path -Path $specOutDirs.Server -ChildPath "Sparrow.dll" ) $versionObj
-        Validate-AssemblyVersion $(Join-Path -Path $specOutDirs.Server -ChildPath "Voron.dll" ) $versionObj
-        Validate-AssemblyVersion $(Join-Path -Path $specOutDirs.Rvn -ChildPath "rvn.dll" ) $versionObj
-        Validate-AssemblyVersion $(Join-Path -Path $specOutDirs.Migrator -ChildPath "Raven.Migrator.dll" ) $versionObj
-        Validate-AssemblyVersion $(Join-Path -Path $specOutDirs.Drtools -ChildPath "Voron.Recovery.dll" ) $versionObj
-        Validate-AssemblyVersion $(Join-Path -Path $specOutDirs.Debug -ChildPath "Raven.Debug.dll" ) $versionObj
+        #Validate-AssemblyVersion $(Join-Path -Path $specOutDirs.Server -ChildPath "Raven.Server.exe" ) $versionObj
```

Comment:
This is still required, no?

Reply:
I will have to check how to do it, but this is now throwing exception.